contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -47,8 +47,10 @@ WORKDIR /tmp
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
       curl ca-certificates xz-utils binutils && \
-    curl -sS -L https://github.com/containernetworking/plugins/releases/download/v0.7.5/cni-plugins-${ARCH}-v0.7.5.tgz -o cni.tar.gz && \
-    tar -xvf cni.tar.gz ./loopback && \
+    curl -sS -L https://github.com/containernetworking/plugins/releases/download/v0.8.6/cni-plugins-linux-${ARCH}-v0.8.6.tgz -o cni-plugins-linux-${ARCH}-v0.8.6.tgz && \
+    curl -sS -L https://github.com/containernetworking/plugins/releases/download/v0.8.6/cni-plugins-linux-${ARCH}-v0.8.6.tgz.sha512 -o cni-plugins-linux-amd64-v0.8.6.tgz.sha512 && \
+    sha512sum -c cni-plugins-linux-${ARCH}-v0.8.6.tgz.sha512 && \
+    tar -xvf cni-plugins-linux-${ARCH}-v0.8.6.tgz ./loopback && \
```

Comment:
Maybe worth adding `ARG CNI_PLUGINS_VERSION=0.8.6` or similar above and use that here to avoid repeating the version several times?

Reply:
Hmm this would imply that we could change the CNI plugin version with `docker build` command which could result in an image with a different CNI version that we were expecting. :thinking:

Reply:
Sorry, I meant `ENV` not `ARG`. But in any case, probably not worth respinning since I assume that version is something that's seldomly bumped.